### ISSUE TITLE
Enable Custom Text for Closed Cells

### ISSUE THREAD

**Issue by MacOS** [nm-wu/ordo#1]
![hidden](https://user-images.githubusercontent.com/3662183/128173107-6cbf5ca6-c3df-4b89-a439-68e022a06c00.png)

Adding custom text for closed cells would be a nice feature as it would signal to the user what is hidden (see picture above).

A major use case is when you have different kings of hidden cells. For example

- notes,
- more detailed elaborations, and
- solutions.

**Comment by MacOS** [comment]
Quick fix without code changes: Insert a cell above the hidden cell.

**Comment by mrcalvin** [comment]
Yes, for now, let's stay with the quick fix. We can think in a number of directions:

- Different admonition types incl. different styling?
- Some sidecar text for the admonition control buttons?
- Different button modes with varying labels: Open -> Close, Close -> Open ...
